{{cookiecutter.project_shortname}}/.travis.yml

```diff
@@ -33,6 +33,8 @@ env:
   matrix:
     - REQUIREMENTS=prod E2E=yes
     - REQUIREMENTS=qa
+    - REQUIREMENTS=devel EXTRAS=all,elasticsearch5,postgresql ES_VERSION=5.6.4
+    - REQUIREMENTS=devel EXTRAS=all,elasticsearch6,mysql ES_VERSION=6.2.2
```

Comment:
I think this has to be parameterized according to which ES version was selected, however I think we shouldn't add ``devel`` requirements in the instance (just have only ``qa`` and ``prod``).